Fedora 19: WARNING: 'aclocal-1.14' is missing on your system.

While sipp-3.4.1 compiles well on Fedora 19 and Oracle EL 6.5, the repo head revision still compiles well on  Oracle EL 6.5 but the same command shows the following error on Fedora 19:

```
# git clone git@github.com:SIPp/sipp.git
# cd sipp
# ./configure --with-openssl --with-pcap --with-rtpstream --with-sctp --with-gsl 
# make
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /home/me/src/sipp/sipp/missing aclocal-1.14 
/home/me/src/sipp/sipp/missing: line 81: aclocal-1.14: command not found
WARNING: 'aclocal-1.14' is missing on your system.
         You should only need it if you modified 'acinclude.m4' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'aclocal' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
         It also requires GNU Autoconf, GNU m4 and Perl in order to run:
         <http://www.gnu.org/software/autoconf>
         <http://www.gnu.org/software/m4/>
         <http://www.perl.org/>
make: *** [aclocal.m4] Error 127
# 
```

Reason is that Fedora 19 ships with version 1.13 and not with the expected 1.14:

```
# rpm -qf /usr/bin/aclocal
automake-1.13.4-1.fc19.noarch
#
```

--- a/README.md
+++ b/README.md
@@ -11,14 +11,14 @@ You should have received a copy of the GNU General Public License along with thi
 
 This is the SIPp package. Please refer to the [webpage](http://sipp.sourceforge.net/) for details and documentation.
 
-Normally, you should be able to build SIPp by just typing "./configure
---with-pcap --with-sctp; make" in the current directory. Then "sipp
--h" will give you access to the online help.
+Normally, you should be able to build SIPp by using the provided build script: `build.sh`. This:
+* checks out the gtest submodule
+* updates the timestamps on the build files, to avoid dependency issues
+* runs configure
+* builds and runs the test suite
+* builds SIPp
 
-Some users have experienced problems with the pre-built autoconf-generated 'configure' file. There are two solutions:
-
-* If you checked out the source through Git, you may need to `touch configure.ac aclocal.m4 configure Makefile.am Makefile.in` to correct the timestamps before running configure and make.
-* You can also rebuild the 'configure' file from scratch by running `autoreconf -ivf`. You will need `autoconf` and `autoconf-archive` installed to make this work.
+`build.sh` passes its arguments through to the configure script, so you can enable SSL, PCAP and SCTP support by calling `./build.sh --with-pcap --with-sctp --with-openssl`.
 
 # Support
 
@@ -27,6 +27,8 @@ I try and be responsive to issues raised on Github, and there's [a reasonably ac
 # Contributing
 
 SIPp is free software, under the terms of the GPL licence (see the LICENCE.txt file for details). You can contribute to the development of SIPp and use the standard Github fork/pull request method to integrate your changes integrate your changes. If you make changes in SIPp, *PLEASE* follow a few coding rules:
+
+  - SIPp uses GNU autotools, so changes to the build process should be done by editing configure.ac and Makefile.in, then regenerating the files with `autoreconf -ivf`. (You will need your distribution's `autotools` and `autoconf-archive` packages installed for this.)
 
   - Please stay conformant with the current indentation style (4 spaces
     indent, standard Emacs-like indentation). Examples:

--- a/build.sh
+++ b/build.sh
@@ -1,0 +1,6 @@
+git submodule update --init &&
+touch configure.ac aclocal.m4 configure Makefile.am Makefile.in &&
+./configure $@ &&
+make sipp_unittest &&
+./sipp_unittest &&
+make

